www/includes/easyparliament/templates/html/alert/index.php

```diff
@@ -374,6 +381,12 @@
                         </li>
                       <?php } ?>
                     </ul>
+
+                    <form action="<?= $actionurl ?>" method="POST" class="pull-right">
+                        <input type="hidden" name="t" value="<?= _htmlspecialchars($alert['token']) ?>">
+                        <input type="submit" class="button button--negative small" name="action" value="Delete All">
+                    </form>
```

Comment:
This doesn't interact well with the current mp section below:

![screenshot 2018-12-07 at 16 24 31](https://user-images.githubusercontent.com/5310/49659701-b404df00-fa3c-11e8-9716-a6d3bd4295a4.png)

Reply:
Whoops, should be okay now. Also is now identical on /user/ page.